### PR TITLE
RavenDB-7721 - marked obsolete fields + using configuration props whe…

### PR DESCRIPTION
**Pull request by ml054**
…rever possible

### CODE REVIEW

File: src/Raven.Studio/typescript/commands/database/documents/getPeriodicExportSetupCommand.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import database = require("models/resources/database");
 class getPeriodicExportSetupCommand extends getDocumentWithMetadataCommand {
 
     constructor(db: database) {
-        super("Raven/PeriodicExport/Configuration", db, true);
+        super("Raven/PeriodicExport/Configuration", db, true); //TODO: no longer used?
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/database/documents/getPeriodicExportSetupCommand.ts:8]: IIRC, this is now solely in the `DatabaseRecord`, no?

File: src/Raven.Studio/typescript/commands/database/globalConfig/getGlobalConfigReplicationsCommand.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import database = require("models/resources/database");
 class getGlobalConfigReplicationsCommand extends getDocumentWithMetadataCommand {
 
     constructor(db: database) {
-        super("Raven/Global/Replication/Destinations", db);
+        super("Raven/Global/Replication/Destinations", db); //TODO: obsolete?
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/database/globalConfig/getGlobalConfigReplicationsCommand.ts:7]: This is dead, yes

File: src/Raven.Studio/typescript/commands/database/replication/getAutomaticConflictResolutionDocumentCommand.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import database = require("models/resources/database");
 class getAutomaticConflictResolutionDocumentCommand extends getDocumentWithMetadataCommand {
 
     constructor(db: database, global: boolean  = false) {
-        super(global ? "Raven/Global/Replication/Config":"Raven/Replication/Config", db);
+        super(global ? "Raven/Global/Replication/Config" : "Raven/Replication/Config", db); //TODO: obsolete?
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/database/replication/getAutomaticConflictResolutionDocumentCommand.ts:7]: Dead

File: src/Raven.Studio/typescript/commands/database/replication/getReplicationSourcesCommand.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ class getReplicationSourcesCommand extends commandBase {
 
     execute(): JQueryPromise<dictionary<string>> {
         var args = {
-            startsWith: "Raven/Replication/Sources",
+            startsWith: "Raven/Replication/Sources", //TODO: obsolete?
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/database/replication/getReplicationSourcesCommand.ts:11]: Dead

File: src/Raven.Studio/typescript/commands/database/replication/saveAutomaticConflictResolutionDocumentCommand.ts [hunk 5]
```diff
@@ -15,7 +15,7 @@ class saveAutomaticConflictResolutionDocumentCommand extends commandBase {
     }
 
     private saveConfig(): JQueryPromise<any> {
-        var id = this.globalConfig ?  "Raven/Global/Replication/Config" : "Raven/Replication/Config";
+        var id = this.globalConfig ? "Raven/Global/Replication/Config" : "Raven/Replication/Config"; //TODO: obsolete?
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/database/replication/saveAutomaticConflictResolutionDocumentCommand.ts:18]: Dead

File: src/Raven.Studio/typescript/commands/database/replication/saveReplicationDocumentCommand.ts [hunk 6]
```diff
@@ -15,7 +15,7 @@ class saveReplicationDocumentCommand extends commandBase {
     }
 
     private saveSetup(): JQueryPromise<any> {
-        var id = this.globalConfig ? "Raven/Global/Replication/Destinations" : documentReplicationConfiguration;
+        var id = this.globalConfig ? "Raven/Global/Replication/Destinations" : documentReplicationConfiguration; //TODO: obsolete
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/database/replication/saveReplicationDocumentCommand.ts:18]: Dead

File: src/Raven.Studio/typescript/commands/maintenance/backupDatabaseCommand.ts [hunk 7]
```diff
@@ -33,7 +33,7 @@ class backupDatabaseCommand extends commandBase {
     }
 
     private getBackupStatus(result: JQueryDeferred<any>) {
-        new getDocumentWithMetadataCommand("Raven/Backup/Status", this.db)
+        new getDocumentWithMetadataCommand("Raven/Backup/Status", this.db) //TODO: obsolete
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/maintenance/backupDatabaseCommand.ts:36]: In `DatabaseRecord`

File: src/Raven.Studio/typescript/commands/maintenance/monitorCompactCommand.ts [hunk 8]
```diff
@@ -7,7 +7,7 @@ class monitorCompactCommand extends commandBase {
     }
 
     execute(): JQueryPromise<any> {
-        new getDocumentWithMetadataCommand("Raven/Database/Compact/Status/" + this.dbName, null)
+        new getDocumentWithMetadataCommand("Raven/Database/Compact/Status/" + this.dbName, null) //TODO: obsolete?
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/maintenance/monitorCompactCommand.ts:10]: I'm not sure, right now I don't think it is supported at all, at any rate, it wouldn't be using this doc.

File: src/Raven.Studio/typescript/commands/maintenance/monitorRestoreCommand.ts [hunk 9]
```diff
@@ -7,7 +7,7 @@ class monitorRestoreCommand extends commandBase {
     }
 
     execute(): JQueryPromise<any> {
-        new getDocumentWithMetadataCommand("Raven/Restore/Status", null)
+        new getDocumentWithMetadataCommand("Raven/Restore/Status", null) //TODO: obsolete
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/maintenance/monitorRestoreCommand.ts:10]: It isn't a document, but we do need to track it, I think @grisha-kotler is already doing it.

File: src/Raven.Studio/typescript/commands/maintenance/startRestoreCommand.ts [hunk 10]
```diff
@@ -10,7 +10,7 @@ class startRestoreCommand extends commandBase {
     execute(): JQueryPromise<any> {
         var result = $.Deferred();
 
-        new deleteDocumentCommand('Raven/Restore/Status', null)//TODO: use endpoints
+        new deleteDocumentCommand('Raven/Restore/Status', null)//TODO: use endpoints, obsolete
```
Comment (ayende) [inline src/Raven.Studio/typescript/commands/maintenance/startRestoreCommand.ts:13]: dead

File: src/Raven.Studio/typescript/models/database/documents/periodicExportSetup.ts [hunk 11]
```diff
@@ -264,10 +264,10 @@ class periodicExportSetup {
 
     fromDatabaseSettingsDto(dbSettingsDto: documentDto) {
         this.dbSettingsDto = dbSettingsDto;
-        this.awsAccessKey(dbSettingsDto["Settings"]["Raven/AWSAccessKey"]);
-        this.awsSecretKey(dbSettingsDto["SecuredSettings"]["Raven/AWSSecretKey"]);
-        this.azureStorageAccount(dbSettingsDto["Settings"]["Raven/AzureStorageAccount"]);
-        this.azureStorageKey(dbSettingsDto["SecuredSettings"]["Raven/AzureStorageKey"]);
+        this.awsAccessKey(dbSettingsDto["Settings"]["Raven/AWSAccessKey"]); //TODO: obsolete
```
Comment (ayende) [inline src/Raven.Studio/typescript/models/database/documents/periodicExportSetup.ts:267]: @grisha-kotler What are you doing here?

File: src/Raven.Studio/typescript/viewmodels/manage/restore.ts [hunk 12]
```diff
@@ -73,7 +73,7 @@ class restore extends viewModelBase {
             this.isBusy(true);
             var self = this;
 
-            new getDocumentWithMetadataCommand("Raven/Restore/InProgress", null, true).execute()
+            new getDocumentWithMetadataCommand("Raven/Restore/InProgress", null, true).execute() //TODO: obsolete
```
Comment (ayende) [inline src/Raven.Studio/typescript/viewmodels/manage/restore.ts:76]: dead

File: src/Raven.Studio/typescript/commands/database/documents/getPeriodicExportSetupCommand.ts (hunk 1 above)
Comment (ml054) [reply src/Raven.Studio/typescript/commands/database/documents/getPeriodicExportSetupCommand.ts:8]: I assume @grisha-kotler will clean it up as part of his work on backup/restore.

File: src/Raven.Studio/typescript/commands/database/globalConfig/getGlobalConfigReplicationsCommand.ts (hunk 2 above)
Comment (ml054) [reply src/Raven.Studio/typescript/commands/database/globalConfig/getGlobalConfigReplicationsCommand.ts:7]: we clean it up once replication will be finished.

File: src/Raven.Studio/typescript/commands/maintenance/monitorCompactCommand.ts (hunk 8 above)
Comment (ml054) [reply src/Raven.Studio/typescript/commands/maintenance/monitorCompactCommand.ts:10]: statuses will be held in notification center operations.

**Pull request merged**